ci/run-all.sh

```diff
@@ -11,7 +11,8 @@ RUN_DOCKER="${BASE_DIR}/ci/run-docker.sh"
 
 export RUST_VERSION=1.7.0
 
-export DOCKER_IMAGE=posborne/rust-cross:base
+export DOCKER_IMAGE=posborne/rust-cross:x86
+RUST_TARGET=i686-unknown-linux-gnu ${RUN_DOCKER}
```

Comment:
This didn't work previously but does not because the `x86` image includes `gcc-multilib` now so we can run i686 binaries.